src/main/main.cpp

```diff
@@ -440,14 +440,14 @@ int main(int argc, const char *argv[]) {
         return;
       }
 
+	  std::unique_ptr<uint8_t[]> row{new uint8_t[output.width() * 3]};
```

Comment:
Can you use spaces and not tabs please? Thanks!
I'd prefer not having naked `new` in the code (I realise it's safe in this context though): perhaps using a `vector<uin8t_t>` here? I'm  not too worried though, just an observation.